sp/src/game/server/hl2/npc_manhack.cpp

```diff
@@ -470,8 +478,11 @@ void CNPC_Manhack::TakeDamageFromPhyscannon( CBasePlayer *pPlayer )
 
 	// Convert velocity into damage.
 	Vector vel;
-	VPhysicsGetObject()->GetVelocity( &vel, NULL );
-	float flSpeed = vel.Length();
+	float flSpeed = 0;
+	if (VPhysicsGetObject()) {
```

Comment:
Do manhacks not have VPhysics objects while held by metrozombies?

Reply:
VPhysicsGetObject() returns NULL while they are held. Basically the zombine grenade (or in this case, the manhack) has its physics object destroyed while it is being held and then re-initialized when it is dropped